Documentation/Blazorise.Docs/Pages/Docs/Extensions/Video/VideoApi.razor

```diff
@@ -146,4 +146,7 @@
     <DocsAttributesItem Name="SettingsList" Type="VideoSettingsType[]" Default="Captions, Quality, Speed, Loop">
         If the default controls are used, you can specify which settings to show in the menu.
     </DocsAttributesItem>
+    <DocsAttributesItem Name="ClickToFullscreen" Type="bool" Default="true">
+        Double-press the video container to toggle fullscreen.
```

Comment:
Is it double press or single click? I mean it should be fairly obvious that a regular click will not full screen it, however to make sure we're explicit and don't leave room for misinterpretations, shouldn't the name of the parameter better reflect this?

"`DoubleClickToFullScreen`?"